ci-operator/jobs/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master-presubmits.yaml

```diff
@@ -1,5 +1,72 @@
 presubmits:
   openshift/cloud-provider-openstack:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build02
+    context: ci/prow/e2e-openstack-ipi
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-openstack-master-e2e-openstack-ipi
```

Comment:
You might want to add “optional: true” here until the test is reliable

Reply:
I would prefer to keep this job mandatory. I don't want to merge anything if it is red.